Library/Homebrew/keg.rb

```diff
@@ -217,6 +217,10 @@ def empty_installation?
     true
   end
 
+  def require_relocation?
+    # no-op
```

Comment:
Actually had `false` in there before but thought a comment would be even *no-op*-ier. 😂